types/webpack-blocks__core/webpack-blocks__core-tests.ts

```diff
@@ -0,0 +1,21 @@
+import { createConfig, group, env, match, when, Block, Context, MatchOptions, Util } from '@webpack-blocks/core';
```

Comment:
Is the import correct? The documentation only uses `webpack-blocks` and not `@webpack-blocks/core`.

Reply:
If you check [doc](https://www.npmjs.com/package/webpack-blocks) you will see following line: "This is the webpack-blocks convenience package. It wraps all the most commonly used blocks, so you can install just this single package and have webpack-blocks and your favorite blocks set up."

My plan:
1. Cover separate blocks (@webpack-blocks/core and so on)
2. Create main package with links to separate packages.

If we cover all functions in one package (webpack-blocks), we will leave without support subpackages (@webpack-blocks/core, @webpack-blocks/webpack and so on).

Reply:
Technically, I can do this in one pull-request:
1. create several folders for (i.e. webpack-blocks__core and so on).
2. create one folder for webpack-blocks and import separate blocks.